src/main/java/http/ResponseHandler.java

```diff
@@ -2,8 +2,17 @@
 
 public class ResponseHandler {
 
-    public byte[] getPageNotFoundResponse() {
-        String pageNotFoundResponse = "HTTP/1.1 404 Not Found\r\n\r\n";
-        return pageNotFoundResponse.getBytes();
+    public ResponseHandler() {
```

Comment:
If this is empty, consider making this a static class.

Reply:
Amended to a static class